inst/rmd/h/anchor-sections/anchor-sections.css

```diff
@@ -0,0 +1,21 @@
+@media (min-width: 1000px) {
+  h1:hover a.anchor,
+  h2:hover a.anchor,
+  h3:hover a.anchor,
+  h4:hover a.anchor,
+  h5:hover a.anchor,
+  h6:hover a.anchor {
+    visibility: visible;
+  }
+}
+a.anchor {
+  margin-left: -30px;
+  display: inline-block;
+  width: 30px;
+  height: 30px;
+  visibility: hidden;
+  background-image: url(./link.svg);
```

Comment:
Sorry, I'm a minimalist. How about just using the character `#` instead of the SVG icon? Like:

```css
a.anchor::before {
  content: "#";
}
```

If users prefer using icons, they could override the CSS rules by themselves.

Reply:
I love the minimalism, too! Reflected the review via [`8579032` (#1884)](https://github.com/rstudio/rmarkdown/pull/1884/commits/8579032206d353948c7e63048cc60fdc2bec2202)